### ISSUE TITLE
[1.7.10] Crash with conduits and a turbine

### ISSUE THREAD

**Issue by Eufranio** [SleepyTrousers/EnderIO#3742]
https://github.com/aidancbrady/Mekanism/issues/3850

**Comment by DaMachinator** [comment]
I so happen to have both mods so I'll try to replicate this.

**Comment by Eufranio** [comment]
There is no errors beyond that, when a player loads the chunk with the conduit the whole server crashes. I'm using both latest Mekanism and EnderIO (from curse), on a 1.7.10 server. The only log/report from the error is that crash.

**Comment by VT-14** [comment]
Here is the template you deleted when you opened this issue, without an (in this case) unnecessary modpack note.  Also include Mekanism's version number.  Also, post the entire crash logs (in pastebin or a Github Gist).  The stacktrace you posted is quite often not enough to diagnose the issue; crash logs aren't insanely long just to look pretty.

For instance, the crash log might pinpoint the location of the turbine or conduit having issues, in which case you might be able to use commands to set the problem block to a generic stone block to get that area of the server working again.

You should have a both a crash log and generic log from the Server.

Until the following template is filled out, this issue will remain marked "Logfile Missing" and "Report Incomplete" and you will likely be ignored by the developers.

#### Issue Description:



#### What happens:



#### What you expected to happen:



#### Steps to reproduce:

1.
2.
3.
...

____
#### Affected Versions (Do *not* use "latest"):

- EnderIO:
- EnderCore:
- Minecraft: 
- Forge:

#### Your most recent log file where the issue was present: 

[pastebin/gist/etc link here]

**Comment by HenryLoenwind** [comment]
Btw, I looked at the posted stack trace. The given line number of the crash makes no sense for the latest version of the Mekanism source code. So without knowing which version of Mekanism produced this crash, the stack trace is useless.

**Comment by Eufranio** [comment]
Okay, let me rephrase, sorry... 

**Issue Description:**
A player put a fluid conduit next a Turbine (not Wind turbine, a turbine multiblock that generates energy using steam), broke a block from the turbine (the multiblock structure disappeared) and then the server crashed.
**What happens:**
The server crashed, and still crashing every time that any player loads the chunk with the conduit/turbine.
**What you expected to happen:**
The conduit extract/insert steam into the turbine.
Steps to reproduce:

1. Build a turbine
2. Place a fluid conduit next to the turbine
3. Break a block to break the multiblock

**I tested some times, and 50% of the times it crashed, and 50% not. I don't know what is causing this, I tested the exact setup all the times.**

I didn't include the whole crash log because it's not a **ticking block** crash, and there is no information about where the block is, only that stacktrace, but here is the entiere crash log:

http://pastebin.com/nrxPNk6w

**Versions:**

EnderIO: 1.7.10 - 2.2.8.381
EnderCore: 1.7.10 - 0.2.0.36_beta
Mekanism: 1.7.10 - 9.0.3.278
MekanismGenerators: 1.7.10 - 9.0.3.278

Modpack / Mod list: http://www.technicpack.net/modpack/blackcraft-server.591976/mods
